gnocchi/rest/aggregates/api.py

```diff
@@ -246,21 +246,32 @@ def post(self, start=None, stop=None, granularity=None,
             except indexer.IndexerException as e:
                 api.abort(400, six.text_type(e))
             if not groupby:
-                return self._get_measures_by_name(
-                    resources, references, body["operations"], start, stop,
-                    granularity, needed_overlap, fill, details=details)
+                try:
+                    return self._get_measures_by_name(
+                        resources, references, body["operations"], start, stop,
+                        granularity, needed_overlap, fill, details=details)
+                except indexer.NoSuchMetric as e:
```

Comment:
Would you mind rather adding a `jsonify` method to the `indexer.NoSuchMetric` exception?

Reply:
sure thing.